codebase/superdarn/src.idl/app/examples.1.2/readcnvmap.pro

```diff
@@ -2,30 +2,24 @@
 ; ==============
 ; Author: R.J.Barnes
 ; 
-; LICENSE AND DISCLAIMER
-; 
+;
 ; Copyright (c) 2012 The Johns Hopkins University/Applied Physics Laboratory
-; 
-; This file is part of the Radar Software Toolkit (RST).
-; 
-; RST is free software: you can redistribute it and/or modify
-; it under the terms of the GNU Lesser General Public License as published by
-; the Free Software Foundation, either version 3 of the License, or
-; any later version.
-; 
-; RST is distributed in the hope that it will be useful,
-; but WITHOUT ANY WARRANTY; without even the implied warranty of
-; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-; GNU Lesser General Public License for more details.
-; 
-; You should have received a copy of the GNU Lesser General Public License
-; along with RST.  If not, see <http://www.gnu.org/licenses/>.
-; 
-; 
-; 
 ;
-; A Simply demonstration program for reading cnvmap files
```

Comment:
It appears the description for each of these IDL example routines was inadvertently deleted when changing the license/copyright header information.

Reply:
Thanks for the catch here @egthomas!  Looks like that might be a whoops back on 7280cf2.  Is there something special with a Simply demo program or is that a typo and might read better as: "A simple demonstration program"?  Not sure if this is something specific to IDL or not.

Reply:
@ksterne agreed that was likely a typo, and your suggested change ("A simple demonstration program for ...") sounds appropriate.

Reply:
Fixed this with https://github.com/SuperDARN/rst/pull/441/commits/a694599068849626e6112ba99550f62d27078fc5